nav/config_files/config_01_settings.yaml

```
@@ -201,15 +204,28 @@ rings:
 ##################
 
 stars:
-  calibrated_data: true
+  catalogs: [ ucac4, tycho2, ybsc ]  # Order of most precise to least
```

YAMLlint (1.37.1)

[error] too many spaces inside brackets

(brackets)